lib/plugins/blocks.js

```diff
@@ -103,6 +105,21 @@ function inject (bot, { version }) {
     bot.emit('chunkColumnLoad', columnCorner)
   }
 
+  function waitForChunksToLoad (cb) {
+    // check 5x5 chunks around us
+    for (let x = -2; x <= 2; x++) {
+      for (let y = -2; y <= 2; y++) {
+        if (bot.blockAt(bot.entity.position.plus(new Vec3(x, 0, y).scaled(16))) == null) {
```

Comment:
Instead of checking the block, it would be best to check the column !== null directly.
Something like:
```js
const chunkX = Math.floor(bot.entity.position.x / 16) + x
const chunkZ = Math.floor(bot.entity.position.z / 16) + z
if (bot.world.getColumn(chunkX, chunkZ) !== null) {
```

Reply:
done